### PR TITLE
Lint out font-face use

### PR DESCRIPTION
**Pull request by gtrufitt**
## What does this change?
- adds a property blacklist for `font-face` so it can't be used in the CSS
- changes the typography export for the font faces to remove complexity and export the entire rule (which matches the `fs()` functions that are exported)

## Why?
As we have pasteup, there should not be a need to use `font-face` and if there is it should be explicit and commented (see ad.tsx).

image

*Note: This is major version bump for pasteup due to incompatibilities of typography export*

cc/ @guardian/dotcom-platform

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -6,6 +6,7 @@
   ],
   "rules": {
     "no-duplicate-selectors": null,
-    "no-descending-specificity": null
+    "no-descending-specificity": null,
+    "property-blacklist": [["font-family"], { "message": "Please use the typography functions from pasteup/typography"}]
```
Comment (SiAdcock) [inline .stylelintrc]: How about adding `font-size` and `line-height` too?

Comment (gtrufitt) [reply .stylelintrc]: Yeah, think that's my plan, `line-height` is slightly more difficult due to usage in vertical centring. But future PR's, doing a rule at a time 👍

File: packages/pasteup/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@guardian/pasteup",
-  "version": "1.0.0-alpha.11",
+  "version": "2.0.0-alpha.0",
```
Comment (nicl) [inline packages/pasteup/package.json]: As we're alpha, we can probably keep it on 1. Our versioning seems a bit weird tbh - we should be pre-version-1 if alpha really.

Comment (gtrufitt) [reply packages/pasteup/package.json]: My concern would be that someone didn’t heed the warning and has used the npm module and a minor update would then break their code. However, I think I can instead use [pre release versioning](https://docs.npmjs.com/misc/semver#prerelease-tags) but need to release properly. Will read up!

Comment (SiAdcock) [reply packages/pasteup/package.json]: ⍺⁴𝙻

**Pull request merged**